R/init.R

```diff
@@ -21,12 +21,16 @@
 .pkgenv <- new.env(parent=emptyenv())
 
 .onAttach <- function(libname, pkgname) {
+    if (getOption("blpVerbose", FALSE)) {
```

Comment:
Yes, I'm in favor of making this unconditional. Not something that needs to be attached to a global option.